src/elements/mixins.pug

```diff
@@ -17,6 +11,6 @@ mixin image(src, alt)
   img(src=src alt=alt)&attributes(attributes)
 
 //- Inputs
-mixin input(label, id, maxlength, regex)
+mixin input(label, id, name, type, maxlength, regex, title)
   label(for=id)= label
-  input(id=id, name=id, type='text' maxlength=maxlength pattern=regex)
\ No newline at end of file
+  input(id=id, name=name, type=type maxlength=maxlength pattern=regex title=title)
```

Comment:
How comes there aren't any commas between type, maxlength, pattern, or title, but there are between id and name?